src/cmakeTools.ts

```diff
@@ -4,7 +4,7 @@
 import { CMakeCache } from '@cmt/cache';
 import { CMakeExecutable, getCMakeExecutableInformation } from '@cmt/cmake/cmake-executable';
 import { CompilationDatabase } from '@cmt/compdb';
-import * as debugger_mod from '@cmt/debugger';
+import * as debuggerMode from '@cmt/debugger';
```

Comment:
I think the `mod` is short for `module`, not `mode`.  Can it be removed?  Just call it `debugger`?

Reply:
debugger was a reserved word, couldn't use it.